src/elements/header.module.scss

```diff
@@ -85,3 +90,15 @@
         }
     }
 }
+
+.hideMobile {
+    @media only screen and (max-width: 600px) {
+        display: none;
+    }
+}
+
+.showMobile {
+    @media only screen and (min-width: 600px) {
+        display: none;
+    }
+}
```

Comment:
`max-width` and `min-width` are both inclusive, so at 600px screen width, both `showMobile` and `hideMobile` will be hidden.